DEVELOPING.md

```diff
@@ -4,7 +4,7 @@
 
 1. Everything required to install `truss`
 2. go-bindata for compiling templates into binary `$ go get
-   github.com/jteeuwen/go-bindata`
+   github.com/jteeuwen/go-bindata/...`
```

Comment:
Favor block-style code markup when providing "type this at the command line" type instructions. They stand out more. E.g., `$ echo hello` vs.
```bash
$ echo hello
```